L1Trigger/TrackFindingTMTT/src/MuxHToutputs.cc

```diff
@@ -0,0 +1,155 @@
+//--- Note that the word "link" appearing in the C++ or comments in this class actually corresponds
+//--- to a pair of links in the hardware.
+
+#include "L1Trigger/TrackFindingTMTT/interface/MuxHToutputs.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Settings.h"
+#include "L1Trigger/TrackFindingTMTT/interface/PrintL1trk.h"
+
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include <sstream>
+#include <mutex>
+
+using namespace std;
+
+namespace tmtt {
+
+  namespace {
+    std::once_flag printOnce;
```

Comment:
move to be a `static` variable inside `MuxHToutputs::MuxHToutputs` ?

Reply:
done